governance/groups.md

```diff
@@ -7,32 +7,48 @@ A list of groups and members follows.
 ### Fix live bugs
 ### Committing changes to production server
 ## Quality review groups
-A list of groups and members follows.
+A list of groups and members follows. The wording of this text will be subject to revision as necessary as determined by the members of the teams upon introduction into the governance documents.
```

Comment:
Again this needs more information and review from other members before it is been added to the governance process. Please follow the process, answer the respective questions.

#53

Thanks.

This comments applies to all the changes below.

Reply:
It was. You were not there.

Reply:
I'm very very sick of this attitude and immaturity you have shown in this comment. You know that I'm in Brussels attending the event and presenting Daemo. I could not attend the meetings. I didnt see you in many other meetings when decisions needed to make. Besides that I mention I will take a look as soon as I get time. Did you answer the questions that were raised? You didn't.